apps/slpquiet/widget.js

```diff
@@ -0,0 +1,36 @@
+(function() {
+  WIDGETS["qmsched"] = {
```

Comment:
i see, this would lead to conflicts when using both apps. renamed the widget to "slpquiet".
Question: what about splitting the widget from (both) app(s), and include it as a dependency in a future version?

Reply:
Cool, thank you. I think the widget is fine as part of `slpquiet`, and now I've got my head around it more, we'll want to make `app.js` a settings file so it's picked up by the Bangle settings app, and likewise for `thmswtch`.

This is quite easy to do, if you could rename `app.js` to `settings.js` for both of them, then in both apps' `metadata.json` files (we'll want to add one for `thmswtch`, we can apply something like:
```diff
-    {"name":"slpquiet.app.js","url":"app.js"},
+    {"name":"slpquiet.settings.js","url":"settings.js"},
```

Then we get `slpquiet` and `thmswtch` settings appearing in Bangle settings for free :)

Reply:
will play with it and implement in near future. thanks for the suggestion! :)

Reply:
tried it for the theme switcher, but it will require some modifications in the menu items etc. so will prepare it in the next version. thanks

Reply:
No worries, sounds good :)